translations/fr_FR/LC_MESSAGES/acsengine.po

```diff
@@ -107,12 +107,12 @@ msgstr "échec d'initialisation du générateur de modèles : %s"
 #, c-format
 msgid "invalid version %s for conversion back from unversioned object"
 msgstr ""
-"version non valide %s pour la reconversion à partir de l'objet sans version"
+"version non valide %s pour la reconversion à partir de l'objet non versionné"
 
 #: pkg/acsengine/engine.go:322
 #, c-format
 msgid "orchestrator '%s' is unsupported"
-msgstr "orchestrator « %s » n'est pas pris en charge"
+msgstr "l'orchestrateur « %s » n'est pas pris en charge"
```

Comment:
@CecileRobertMichon Thoughts on the definite article addition here? ;)

Reply:
The l' works although I'm not sure "orchestrateur" is a thing

Reply:
Nevermind, apparently it is (source: http://www.journaldunet.com/solutions/cloud-computing/1197423-kubernetes-l-orchestrateur-de-containers-qui-passe-a-l-echelle-mondiale/)

Reply:
If I may, "l'orchestrateur « %s » n'est pas compatible" might be a better translation since "pris en charge" usually means paid for.

Reply:
Ah, we got additional language expert here for French!